tests/y-xml.tests.js

```diff
@@ -133,3 +133,21 @@ export const testInsertafter = tc => {
     el.insertAfter(deepsecond1, [new Y.XmlText()])
   })
 }
+
+
+/**
+ * @param {t.TestCase} tc
+ */
+ export const testClone = tc => {
+  const ydoc = new Y.Doc()
+  const yxml = ydoc.getXmlFragment()
+  const first = new Y.XmlText('text')
+  const second = new Y.XmlElement('p')
+  const third = new Y.XmlElement('p')
+  yxml.push([first, second, third])
+  t.compareArrays(yxml.toArray(), [first, second, third])
+
+  const cloneYxml = yxml.clone()
+  t.assert(cloneYxml.length === 3)
+  t.assert(cloneYxml.toJSON() === yxml.toJSON())
```

Comment:
@dmonad, thanks for merging. I must have been asleep at the wheel here. I don't know why I was testing for strict equality here. I'm pretty sure we just wanted the object contents to be equal.

Reply:
haha, no problem ^^ 
Thanks for debugging and providing a PR :)